backend/RoomPlanner/room-planner/src/main/java/edu/roomplanner/entity/ReservationEntity.java

```diff
@@ -0,0 +1,39 @@
+package edu.roomplanner.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.Calendar;
+
+@Entity
+@Table(name = "reservations")
+@Setter
+@Getter
+public class ReservationEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq_reservation_id")
+    @SequenceGenerator(name = "seq_reservation_id", sequenceName = "seq_reservation_id", initialValue = 1, allocationSize = 1)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "start_date", nullable = false)
+    @Temporal(TemporalType.TIMESTAMP)
+    private Calendar startDate;
+
+    @Column(name = "end_date", nullable = false)
+    @Temporal(TemporalType.TIMESTAMP)
+    private Calendar endDate;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "user_id", nullable = false)
+    private Long userId;
```

Comment:
You should have here the user entity for a easy access while developing.
Same for room.
And in the database to store just the id's.